src/kbmod_wf/workflow_tasks/kbmod_search.py

```diff
@@ -36,7 +36,7 @@ def kbmod_search(inputs=(), outputs=(), runtime_config={}, logging_file=None):
     with ErrorLogger(logger):
         kbmod_search(
             wu_filepath=inputs[0].filepath,
```

Comment:
Does this need to be `inputs[0].result().filepath`, since it's being passed as a Future?

Reply:
Hmm I'm not sure if it's the correct practice but the reason why the workflow is working is that it has a DataFuture which has a `filepath` property that we specified earlier (the source code pulling from the parsl File object we defined at initialization) https://parsl.readthedocs.io/en/stable/stubs/parsl.app.futures.DataFuture.html

We could use a call to `result()` to block on the other task completing, but of course since this was a future specified as an argument for the function, parsl has handled that for us and we know the task has completed.

Because of that this seems to work, but it wasn't intentional on my part. So good catch!